packages/wrangler/src/traverse-module-graph.ts

```diff
@@ -36,7 +36,7 @@ export default async function traverseModuleGraph(
 	const bundleType = entry.format === "modules" ? "esm" : "commonjs";
 
 	if (modules.length > 0) {
-		logger.info(`Uploading additional modules:`);
+		logger.info(`Attaching additional modules:`);
```

Comment:
Deploying?

Reply:
Well, you see this message when running `dev` now. So I figured "Uploading" wasn't accurate. "Deploying" probably isn't quite right either.

I did consider "Collecting"/"Collected". But whatever — I'm easy. Happy to go with whatever folks think is best.

Reply:
Not too stressed but my take was that, even in dev, we are deploying to workerd...

Reply:
Going to stick with "Attaching". I think you might spook some users if they see "Deploying" when they're running `dev`.